src/GitHub.App/SampleData/SampleViewModels.cs

```diff
@@ -122,12 +122,6 @@ public ReactivePropertyValidator<string> RepositoryNameValidator
             private set;
         }
 
-        public ICommand Reset
```

Comment:
I had to the option to remove or hide this with `new`, I decided to remove it since this was not in use anywhere,